tests/rust/secure_cell.rs

```diff
@@ -555,7 +555,7 @@ mod seal_with_passphrase {
         let message = b"Colorless green ideas sleep furiously".as_ref();
 
         // Message encrypted by PyThemis
-        let encrypted = b"\x00\x01\x01\x41\x0C\x00\x00\x00\x10\x00\x00\x00\x25\x00\x00\x00\x16\x00\x00\x00\x78\x98\x93\x12\xC9\x60\x1E\x22\xD7\xCB\x47\x06\xC9\xEC\x46\xB5\xB5\x9A\xFC\xC8\x3F\x06\x8F\x5B\xBE\x9F\x66\xA6\x40\x0D\x03\x00\x10\x00\x1C\x6D\x16\xFF\x39\xB9\x13\xDF\xC4\x41\x56\x31\x17\xF3\xC4\x05\x28\x15\x13\xA6\x74\x29\x0B\xF0\x5A\xFB\xAC\xD9\x79\x35\x7E\xBA\xD0\x8F\x8C\xA6\x9E\x4E\x83\x2A\x12\xBA\xC7\x59\xAA\xF6\xDF\x62\x8E\xCE\x31\x7C\xCF".as_ref();
+        let encrypted = cell.encrypt(message).unwrap();
         let decrypted = cell.decrypt(&encrypted).unwrap();
```

Comment:
This one is actually intentional and should be kept this way.

There are quite a few tests around that try decrypting some fixed values. The goal here is to ensure that Themis is able to process data encrypted by some different system. This particular test verifies that RustThemis and PyThemis have the same understanding of UTF-16 endianness.

Unfortunately, Rust does not have UTF-16 string literals so you have to spell it out as byte string, hence an approach like in `default_encoding_is_utf8()` is not applicable here. Instead, PyThemis was used to encrypt some data with literally `"🔑"` string interpreted as UTF-16BE, and here we check that RustThemis understands that. Or that was my idea, if I recall it correctly...

So I'd suggest to keep this test as is. That said, it makes sense to add a comment with reasoning here.

Reply:
This test was annoying, because it's the only one that failed, and I couldn't do anything about it. But your suggestion is interesting!